docs/src/implemented-proposals/rent.md

```diff
@@ -18,13 +24,19 @@ Accounts whose balance is insufficient to satisfy the rent that would be due sim
 
 A percentage of the rent collected is destroyed. The rest is distributed to validator accounts by stake weight, a la transaction fees, at the end of every slot.
 
-## Read-only accounts
+Finally, rent collection happens according to the protocol-level account updates like the rent distribution to validators, meaning there is no corresponding transaction for rent deductions. So, rent collection is rather invisible, only implicitly observable by a recent transaction or predetermined timing given its account address prefix.
+
+## Design considerations
+
+### Current design rationale
 
-Read-only accounts are not being charged rent in current implementation.
+Under the preceding design, it is NOT possible to have accounts that linger, never get touched, and never have to pay rent. Accounts are always pay rent exactly once for each epoch.
```

Comment:
Do you think it's worth calling out that executable accounts, "sys vars", and rent exempt accounts don't need to pay rent?

Reply:
Yeah, that's nice! done!